src/include/sof/drivers/sai.h

```diff
@@ -247,15 +249,28 @@
  * configured for an internal bit clock.
  * The division value is (DIV + 1) * 2.
  */
+#ifndef CONFIG_IMX8ULP
 #define SAI_CLOCK_DIV		0x7
+#else
+#define SAI_CLOCK_DIV		0x5
```

Comment:
CLOCK_DIV should be 5 on NON-ULP platforms where it was 7 before? I feel like the logic is flipped...